app/src/main/java/ar/com/wolox/android/example/ui/home/news/INewsView.kt

```diff
@@ -0,0 +1,3 @@
+package ar.com.wolox.android.example.ui.home.news
+
+interface INewsView
```

Comment:
Please don't leave interfaces or metods empty. Will be completed in another branch?

Reply:
Added TODO comment